src/index.ts

```diff
@@ -5,11 +5,19 @@ import { GITHUB_EVENT } from 'constants/env';
 import { FALSE } from 'constants/boolean';
 import getCommitEmails from 'helpers/getCommitEmails';
 import formatEmailDomain from 'helpers/formatEmailDomain';
+import filterInvalidEmails from 'helpers/filterInvalidEmails';
 
 async function checkEmail(): Promise<void> {
-  const emailDomainInput = getInput(INPUT.EMAIL_DOMAIN, { required: true });
-  const emailDomain = formatEmailDomain(emailDomainInput);
-  info(`Email domain: ${emailDomain}`);
+  const emailDomainsBox: [] = [];
+  const emailDomainsMain = INPUT.EMAIL_DOMAIN;
+  emailDomainsMain.split(',').forEach((email) => {
+    const pushedEmail: string = getInput(email, { required: true });
+    emailDomainsBox.push(pushedEmail);
+  });
```

Comment:
1. Read again what you are splitting. You are splitting the constant `INPUT.EMAIL_DOMAIN`, not the actual input to the GitHub Action
2. IMO splitting of the input should be happening inside the `filterInvalidEmails` function :slightly_smiling_face: This is because we designed the first parameter of that function to be a `string` (`emailDomainInput: string`)